src/wallet/wallet-generation.ts

```diff
@@ -4,6 +4,7 @@ import {RippleAPI} from '..'
 import {errors} from '../common'
 import {GeneratedAddress} from '../offline/generate-address'
 import {isValidAddress} from '../common/schema-validator'
+import { RippledError } from '../common/errors'
```

Comment:
```suggestion
import {RippledError} from '../common/errors'
```
Just making the spacing a little more consistent here